spec/jobs/send_periodic_invite_job_spec.rb

```diff
@@ -51,6 +58,19 @@
         subject
       end
 
+      context "when no creneaux are avaialble" do
+        before do
+          allow(RdvSolidaritesApi::RetrieveCreneauAvailability).to receive(:call).and_return(
+            OpenStruct.new(success?: false, creneau_availability: false)
+          )
+        end
+
+        it "does not send periodic invites" do
+          expect_any_instance_of(described_class).not_to receive(:send_invitation)
```

Comment:
Pour rester homogène 
```suggestion
          expect(Invitations::SaveAndSend).not_to receive(:call)
```